locations/spiders/the_barre_code_us.py

```
@@ -1,39 +1,21 @@
-        "extras": Categories.GYM.value,
-    }
-    allowed_domains = ["thebarrecode.com"]
-    start_urls = ["https://thebarrecode.com/locations/"]
-    rules = [
-        Rule(
-            LinkExtractor(restrict_xpaths='//ul[@id="maplocationsectionbottom"]//a[contains(text(), "Visit Site")]'),
-            callback="parse",
-        )
-    ]
+    item_attributes = {"brand": "The Barre Code", "brand_wikidata": "Q118870170"}
+    start_urls = ["https://thebarrecode.com/"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)